lib/jekyll-import/importers/joomla3.rb

```diff
@@ -16,6 +16,7 @@ def self.specify_options(c)
         c.option 'host', '--host', 'Database host name'
         c.option 'category', '--category', 'ID of the category'
         c.option 'prefix', '--prefix', 'Table prefix name'
+        c.option 'state', '--state', 'Which publishing state to select. 0: unpublished, 1: published, 2: archived (default: 1)'
```

Comment:
This will need to change to `state STATE` right after `c.option` so the parser knows it takes an arg. Surprised the ones above aren't like that... Whoops!